app/models/link.rb

```diff
@@ -1,6 +1,5 @@
 class Link < ApplicationRecord
   before_update :set_time_and_status_on_updated_link, if: :url_changed?
-  after_save :trigger_recount_of_broken_links, if: :status_changed?
```

Comment:
Something we just thought of is that when an individual link is updated, without this `after_save` callback the broken link recount will not occur, and therefore the broken link count will be incorrect.  We should add some tests for this and figure out how to best trigger this recount for individually edited links.